GOVERNANCE.md

```diff
@@ -0,0 +1,146 @@
+
+Maintainers are often under-appreciated, because their work is harder to appreciate.
+It's easy to appreciate a really cool and technically advanced feature. It's harder
+to appreciate the absence of bugs, the slow but steady improvement in stability,
+or the reliability of a release process. But those things distinguish a good
+project from a great one.
+
+## Reviewers
+
+A reviewer is a core role within the project.
+They share in reviewing issues and pull requests and their LGTM counts towards the
+required LGTM count to merge a code change into the project.
+
+Reviewers are part of the organization but do not have write access.
+Becoming a reviewer is a core aspect in the journey to becoming a maintainer.
+
+## Adding maintainers
+
+Maintainers are first and foremost contributors that have shown they are
+committed to the long term success of a project. Contributors wanting to become
+maintainers are expected to be deeply involved in contributing code, pull
+request review, and triage of issues in the project for more than three months.
+
+Just contributing does not make you a maintainer, it is about building trust
+with the current maintainers of the project and being a person that they can
+depend on and trust to make decisions in the best interest of the project.
+
+Periodically, the existing maintainers curate a list of contributors that have
+shown regular activity on the project over the prior months. From this list,
+maintainer candidates are selected and proposed on the maintainers mailing list.
+
+After a candidate has been announced on the maintainers mailing list, the
+existing maintainers are given five business days to discuss the candidate,
+raise objections and cast their vote. Votes may take place on the mailing list
+or via pull request comment. Candidates must be approved by at least 66% of the
+current maintainers by adding their vote on the mailing list. The reviewer role
+has the same process but only requires 33% of current maintainers. Only
+maintainers of the repository that the candidate is proposed for are allowed to
+vote.
+
+If a candidate is approved, a maintainer will contact the candidate to invite
+the candidate to open a pull request that adds the contributor to the
+MAINTAINERS file. The voting process may take place inside a pull request if a
+maintainer has already discussed the candidacy with the candidate and a
+maintainer is willing to be a sponsor by opening the pull request. The candidate
+becomes a maintainer once the pull request is merged.
+
+## Adding sub-projects
+
+Similar to adding maintainers, new sub projects can be added to OpenEBS
+GitHub organization as long as they adhere to the OpenEBS vision and mission. 
+New projects are discussed in either the Contributor Meeting or the Community 
+slack and requiers atleast 1 maintainer approval.
+
+If a project is approved, a maintainer will add the project to the OpenEBS
+GitHub organization, and make an announcement on a public forum.
+
```

Comment:
@kmova , not sure if this is a correct question: On the same lines as above.. should we discuss the processes for addition/removal of features, roadmap items and other such as part of this doc OR is the scope of the governance doc strictly about repo maintenance with feature queries going into community docs.

Reply:
@ksatchit - project management seems to be not included in the other sample governance docs I went through so far. But we could actually make it part of our doc by raising a PR to this doc. I have updated the contributor doc with details on the link to the community meetings page where we discuss about the features in the upcoming releases.

Reply:
Thanks @kmova